d4s2_api/test_serializers.py

```diff
@@ -39,88 +41,46 @@ def test_serializes_delivery_prevents_to_user_in_share_users(self):
         self.assertFalse(serializer.is_valid())
         self.assertIn(SHARE_USERS_INVALID_MSG, serializer.errors['non_field_errors'])
 
+    def test_serializes_share_user_ids_from_db(self):
+        delivery = Delivery.objects.create(project_id='projectA',
+                                           from_user_id='user1',
+                                           to_user_id='user2',
+                                           transfer_id='123-123')
+        DeliveryShareUser.objects.create(delivery=delivery, dds_id='user3')
+        DeliveryShareUser.objects.create(delivery=delivery, dds_id='user4')
+        mock_request = MagicMock()
+        serializer = DeliverySerializer(delivery, context={'request': mock_request})
+
+        self.assertEqual(serializer.data['project_id'], 'projectA')
+        self.assertEqual(serializer.data['from_user_id'], 'user1')
+        self.assertEqual(serializer.data['to_user_id'], 'user2')
+        self.assertEqual(serializer.data['transfer_id'], '123-123')
+        self.assertEqual(serializer.data['share_user_ids'], ['user3','user4'])
+
     def test_finds_related_project(self):
-        p = DukeDSProject.objects.create(project_id=self.data['project_id'])
         serializer = DeliverySerializer(data=self.data)
         self.assertTrue(serializer.is_valid())
         delivery = serializer.save()
-        self.assertEqual(delivery.project, p)
+        self.assertEqual(delivery.project_id, 'project-1234')
 
     def test_finds_related_users(self):
```

Comment:
This test was originally for testing that the serializer created the underlying model relationship. Since there's no underlying model relationship anymore, the name is no longer accurate and I'd argue it's unnecessary since it's only testing a subset of `test_serializes_delivery`